### PR TITLE
Add Go AST Scanner (gas)

### PR DESCRIPTION
**Pull request by tarcieri**
gas is a static analyzer for security issues:

https://github.com/GoASTScanner/gas

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -41,6 +41,15 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          cd $GOPATH/src
+          git clone https://github.com/GoASTScanner/gas.git
+          cd gas
+          go install
```
Comment (kr) [inline wercker.yml]: These four lines could be written as

```
go get github.com/GoASTScanner/gas
```

File: wercker.yml [hunk 2]
```diff
@@ -41,6 +41,15 @@ build:
...
+          cd $GOPATH/src/chain
+          gas
```
Comment (kr) [inline wercker.yml]: When I run just `gas` locally, I get a usage message. Should this be like `gas chain/...` or something?

**Comment by tarcieri** [comment]
@kr this is a WIP (perhaps I should've labeled it as such). At least now I'm getting the same output on Werker that I'm getting locally (presently 42 issues, a mixture of false positives and some legitimate ones)

**Comment by kr** [comment]
Aha, sorry, I jumped the gun there. 😛

**Comment by tarcieri** [comment]
We appear to have hit a regression in gas introducing a false positive for G404: Insecure random number generation. I reported an issue here and disabled the check for now:

https://github.com/GoASTScanner/gas/issues/87

**Pull request closed without merging**